src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Controllers/OverviewController.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.RoatpAssessor.Web.ApplyTypes.Apply;
+using SFA.DAS.RoatpAssessor.Web.Domain;
+using SFA.DAS.RoatpAssessor.Web.Services;
+
+namespace SFA.DAS.RoatpAssessor.Web.Controllers
+{
+    public class OverviewController : Controller
+    {
+        private readonly IAssessorOverviewOrchestrator _overviewOrchestrator;
+
+        public OverviewController(IAssessorOverviewOrchestrator overviewOrchestrator)
+        {
+            _overviewOrchestrator = overviewOrchestrator;
+        }
+
+        [HttpGet("Overview/{applicationId}")]
+        public async Task<IActionResult> ViewApplication(Guid applicationId)
+        {
+            var userId = HttpContext.User.UserId();
+            userId = "temp"; //TODO: Can't access the user until staff idams is enabled
```

Comment:
see previous note about 'temp' - same applies

Reply:
Awaiting iDAMS to work correctly. A ticket will be raised to sort this out.